app/views/projects/index.html.erb

```diff
@@ -1,11 +1,24 @@
-<% @projects.each do |project| %>
-  <h2><%= project.name %></h2>
-    <% project.team_members.each do |team_member| %>
-      <ul>
-        <%= image_tag team_member.thumbnail %>
-        <li><%= team_member.name %></li>
-        <li><%= team_member.role %></li>
-      </ul>
+<div class="row">
+  <% @projects.each do |project| %>
+    <% if project.team_members.count > 0 %>
+      <div class="card card-radius" >
+        <div class="card-body">
+          <h3><%= project.name %></h3>
+            <div class="members">
+            <% project.team_members.each do |team_member| %>
+              <ul class="member unstyled-list">
+                <% if team_member.thumbnail.present? %>
+                  <%= image_tag team_member.thumbnail, style:"width: 50px; border-radius: 50%; " %>
+                <% else %>
+                  <%= image_tag("anon.png", style:"width: 50px; border-radius: 50%; ")  %>
```

Comment:
Let's not style in the template. Use a class instead.

Reply:
I know we shouldn't use inline styling. I researched about how to do a class on an image_tag and I couldn't find an answer. That's why I finally did it in the template, knowing I (or someone who knows better front-end) would need to add a class.